telemetry/Cargo.toml

```diff
@@ -7,11 +7,14 @@ build = "build.rs"
 
 [features]
 dev-telemetry = []
+metric-instrumentation = []
 
 [dependencies]
 iroha_config = { version = "=2.0.0-pre.1", path = "../config" }
 iroha_logger = { version = "=2.0.0-pre.1", path = "../logger" }
 iroha_futures = { version = "=2.0.0-pre.1", path = "../futures", features = ["telemetry"] }
+iroha_telemetry_derive = { version = "=2.0.0-pre.1", path = "derive" }
```

Comment:
can we make this optional since it is strictly `iroha_core` related functionality as it requires `WorldStateView`

Reply:
Nope.